tests/integration/test_api/test_config/test_DOS_blocking_system/test_DOS_blocking_system.py

```diff
@@ -0,0 +1,170 @@
+"""
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: These tests will check if the 'DOS' (Denial-of-service attack) blocking feature of the API handled
+       by the 'wazuh-apid' daemon is working properly. The Wazuh API is an open source 'RESTful' API
+       that allows for interaction with the Wazuh manager from a web browser, command line tool
+       like 'cURL' or any script or program that can make web requests.
+
+components:
+    - api
+
+suite: config
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-apid
+    - wazuh-modulesd
+    - wazuh-analysisd
+    - wazuh-execd
+    - wazuh-db
+    - wazuh-remoted
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/api/getting-started.html
+    - https://documentation.wazuh.com/current/user-manual/api/configuration.html#access
+    - https://en.wikipedia.org/wiki/Denial-of-service_attack
+
+tags:
+    - api
+"""
+import os
```

Comment:
Remove this. Use Path:
```suggestion
from pathlib import Path
```
Apply this in all tests.